chart/templates/replicated-role.yaml

```diff
@@ -31,4 +31,5 @@ rules:
   - {{ include "replicated.secretName" . }}
   - replicated-instance-report
   - replicated-custom-app-metrics-report
+  - replicated-meta-data
```

Comment:
New secret where we store the instance tags.  I gave it a generic name just in case Vendor Portal needed to store other information in the future.

Reply:
from what i see i think this is enough as we have RBAC to `create` secrets and to `update` these specific secrets. i would test that it gets deleted on a `helm uninstall`, but based on what i see that should work since you set the owner reference.

for other secrets we had concern around hitting the 1Mb size limit, but I don't think that is really a concern right now for this one.

the only other thing i would think is "do we want this in a support bundle?". if so, we can add a secret collector to the bundle spec to collect it. we will have the data in airgap support bundles since instance events are written to `replicated-instance-report` and will include these tags. if we think it's valuable to collect the secret data regardless (online or airgap), we can add a collector like [this one](https://github.com/replicatedhq/replicated-sdk/blob/v1.0.0-beta.16/chart/templates/replicated-supportbundle.yaml#L47-L51